proposed improvement (#7)
Joystream/community-repo: README.md

<div align="center">
  <h1>Joystream Community Repo<h1>
</div>
<div align="center">
  <h3>A repo for community reports, tools and contributions.<h3>
</div>

<div align="center">
  <h4>
    <a href="/council-reports">
      Council Reports
    </a>
    <span> | </span>
    <a href="/tokenomics-reports">
    Tokenomics Reports
    </a>
    <span> | </span>
    <a href="/community-contributions">
      Contributions
    </a>
    </a>
  </h4>
</div>

Table of Contents
--

<!-- TOC START min:1 max:3 link:true asterisk:false update:true -->
- [Overview](#overview)
  - [Workflow](#workflow)
    - [KPI Related Submittals](#kpi-related-submittals)
    - [Individual Submittals](#individual-submittals)
    - [Jsgenesis Submittals](#jsgenesis-submittals)
<!-- TOC END -->

# Overview

> The Joystream Community Repo is meant both as a resource for the community members of the Joystream project, and a place to submit their work or contributions.

Reviewer: ```suggestion
The Joystream Community Repo is intended both as a resource for the community members of the Joystream project, and a place to submit their work or contributions.
```

If a KPI requires submitting a deliverable, eg. reports or some code, it is expected that a PR is made to this repo in order to qualify.

Although the community is meant to control the repo, Jsgenesis will approve and merge any pull requests for now. Note that the repo is licensed under [GPLv3](/LICENSE).

## Workflow

The workflow for changing the repo depends on the reason and purpose behind the change.
A consistent part is for the contributor to fork the repo, and create a pull request to the applicable branch.

> ### KPI Related Submittals

Reviewer: ```suggestion
### KPI Related Submissions
```

When a KPI requires a deliverable to be successful, the following steps must be made:
- A pull request is made to the master branch.
- A proposal is made to the [Joystream testnet](https://testnet.joystream.org/).
  - The proposal (`Text`, or in some cases, `Spending`) contains a link to the PR and other relevant information
  - When (if) the proposal is voted through, @bwhm and @blrhc is tagged
  - The time of the latest commit will be used as the time of submission
- The PR is reviewed, and as long as it does not contain anything malicious or does not comply with license, it is merged.

> ### Individual Submittals

Reviewer: ```suggestion
### Individual Submissions
```

**Placeholder: ->**
If the deliverable is made by an individual, eg. for an existing or upcoming funding proposal, the following steps must be made:
- A pull request is made to the community branch?
- Action?
- Merged to community branch?
- PR to master

> - The PR is reviewed, and as long as it does not contain anything malicious or does not comply with license, it is merged.

Reviewer: ```suggestion
- The PR is reviewed, and as long as it does not contain anything malicious or does not comply with the license of the repo, it is merged.
```

**<- Placeholder**

> ### Jsgenesis Submittals

Reviewer: ```suggestion
### Jsgenesis Submissions
```

**Placeholder: ->**

> If a member Jsgenesis wants to make changes to the repo, the following steps must be made:

Reviewer: ```suggestion
If a member of the Jsgenesis team wants to make changes to the repo, the following steps must be taken:
```
